src/Numerics/BasicMath.php

```diff
@@ -0,0 +1,30 @@
+<?php namespace Moserware\Skills\Numerics;
+
+/**
+ * Basic math functions.
+ * 
+ * @author     Jeff Moser <jeff@moserware.com>
+ * @copyright  2010 Jeff Moser 
+ */
+class BasicMath {
```

Comment:
I also prefer this style, but I set my editor to follow the current style to make it consistent - probably just forgot this file.

PSR-2 could be an "enhancement" further down the line. It's currently mostly compliant to PSR-2 and the only big thing is that namespaces shouldn't be declared on the same line as the opening PHP tag as I have done. There's probably a few other things that needs to be double checked as well.